src/ui/public/agg_types/buckets/terms.js

```diff
@@ -7,6 +7,7 @@ import AggTypesBucketsCreateFilterTermsProvider from 'ui/agg_types/buckets/creat
 import orderAggTemplate from 'ui/agg_types/controls/order_agg.html';
 import orderAndSizeTemplate from 'ui/agg_types/controls/order_and_size.html';
 import routeBasedNotifierProvider from 'ui/route_based_notifier';
+import migrateIncludeExcludeFormat from './migrate/include_exclude';
```

Comment:
I wouldn't really separate this into its own file. It seems what we need to test is that the include/exclude params for terms have this new read/write behavior, iso. isolating so we can test this mixin in isolation.